chsdi/locale/empty_chsdi.po

```diff
@@ -8860,3 +8860,33 @@ msgstr ""
 
 msgid "klimaeignung_zone"
 msgstr ""
+
+msgid "tt_eta_ch"
```

Comment:
Can you search the features of this layer?
If yes you need to use the convention `layerId.columnName` as IDs for translations.